src/compile/unit.ts

```diff
@@ -44,7 +44,7 @@ export class UnitModel extends Model {
    */
   private _height: number;
 
-  private readonly _mark: Mark;
+  private readonly _markDef: MarkDef;
```

Comment:
Not sure why we do this weird underscore business. What fi you want to make a variable public in a subclass. 

Feels like an old hacky thing that we needed when we didn't have a type checker.